foundation.cabal

```diff
@@ -137,6 +137,13 @@ Test-Suite test-foundation
   ghc-options:       -Wall -fno-warn-orphans -fno-warn-missing-signatures
   default-language:  Haskell2010
 
+Test-Suite doctest
+  type:              exitcode-stdio-1.0
+  hs-source-dirs:    tests
+  Main-is:           DocTest.hs
+  Build-Depends:     base >= 3 && < 5
+                   , doctest >= 0.11
```

Comment:
in `lts-5.18`:

>  doctest: needed (>=0.11), 0.10.1 found

Do we specifically need, at least, this version of doctest? Another option would be to disable the `doctest` tests for lts-5 in Travis.